### ISSUE TITLE
There is a way to effectively add a custom Error handler to multer?

### ISSUE THREAD

**Issue by AyresMonteiro** [expressjs/multer#1229]
I'm trying to do something like this:

```ts
let multerware = multer({
  limits: { fileSize: 50 * 1024 * 1024 }
}).single("file");

multerware(req, res, (err) => {
  if (err) {
    return res.status(500).json({
      error: true,
      message: "You have exceeded 50MB file size limit. Send a smaller file instead"
    });
  }

  return next();
});
```

But it does not work properly, can someone help me?

**Comment by Doc999tor** [comment]
It's a bit hard to know what exactly you're doing but:
1. If you want to catch `content too large 413` error, you can do it with `limits` key in storage object
2. It's difficult to understand what you're trying to do with `multerware, but if you want to call this middleware it'll interrupt the middleware chain and return the 500 error

**Comment by AyresMonteiro** [comment]
Hi! I figured out that the 413 status code was being returned by other application.

TL;DR.

The presented way in this issue works well and I recommend it.

Thanks!
